conceptual/Aspects/Diagnostics.md

```diff
@@ -35,8 +35,10 @@ To report a diagnostic:
 
 ### Example
 
-The following aspect needs a field named `_logger` to exist in the target type. Its `BuildAspect` method checks for that this field exist and reports an error if it does not.
-
+The following aspect needs a field named `_logger` to exist in the target type. Its `BuildAspect` method checks that this field exists and reports an error if it does not.
+<!--- Ok I know that this is picking hairs, BUT... Thus far in your webinar and other presentation material that you have made you have demonstrated Metalamas ability to produce quick and easy logging, 
```

Comment:
```suggestion
```

I think the example is still relevant. The aspect is a reusable entity in itself and it should check its preconditions, regardless of how it is then added.